webpack/assets/javascripts/react_app/components/componentRegistry.js

```diff
@@ -123,6 +124,7 @@ const coreComponets = [
   { name: 'ConfigReports', type: ConfigReports },
   { name: 'DiffModal', type: DiffModal },
   { name: 'TemplateInput', type: TemplateInput },
+  { name: 'Slot', type: Slot },
```

Comment:
What about `Fill`?

Reply:
Fill is a registration component, therefore it doesn't make sense to add it for erb use (use globalFill instead, or the consumer can create a component with a fill and a desire component to register)